Makefile

```diff
@@ -68,6 +68,14 @@ docs_clean:
 .PHONY: unittest_tests
 unittest_tests: $(TESTS_SRCS)/*_unittest.py
 
+.PHONY: pytest_tests
+pytest_tests: FORCE coverage_erase
+	@printf "${TERM_BRIGHT}PYTEST pytest_testslide${TERM_NONE}"
+	PYTHONPATH=${CURDIR}/pytest-testslide:${CURDIR} \
+	${Q} coverage run \
+	-m pytest \
```

Comment:
pytest is not installed by rule `install_build_deps`, I believe it'll fail if one follows what's on `CONTRIBUTING.md`, right?